bitnami/harbor/templates/trivy/trivy-svc.yaml

```diff
@@ -15,7 +16,7 @@ spec:
   ports:
     - name: api-server
       protocol: TCP
-      port: {{ include "harbor.trivy.servicePort" . }}
+      port: {{ ternary .Values.trivy.containerPorts.https .Values.trivy.containerPorts.http .Values.internalTLS.enabled }}
```

Comment:
Same, check all cases

Reply:
To be resolved in previous conversation.